idaes/models_extra/column_models/tests/test_solvent_condenser.py

```diff
@@ -300,6 +301,9 @@ def model(self):
 
         m.fs.unit.heat_duty.fix(-6264)
 
+        xfrm = TransformationFactory("contrib.strip_var_bounds")
```

Comment:
Could you comment on why this is needed? An inline comment in the code might also be good for future reference.